types/react-portal/v3/react-portal-tests.tsx

```diff
@@ -7,6 +7,9 @@ export default class App extends React.Component {
   render() {
     const button1 = <button>Open portal with pseudo modal</button>;
 
+    // $ExpectError
+    <Portal />;
+
```

Comment:
Wait, shouldn't this be inside the return ( ... ) block?

Reply:
Can't get `ExpectError` to work inside JSX children. What would be the value in having it inside `return`?